bundles/InstallBundle/src/Installer.php

```diff
@@ -730,7 +730,7 @@ public function setupDatabase(Connection $db, array $userCredentials, array $err
 
     protected function getDataFiles(): array
     {
-        return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', \GLOB_BRACE);
+        return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', (\defined('GLOB_BRACE') ? \GLOB_BRACE : 0));
```

Comment:
But does it find anything if we don't set the flag?

Reply:
If not, a solution would be:

```suggestion
        if (\defined('\GLOB_BRACE')) {
            return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', \GLOB_BRACE);
        }
        
        return array_merge(
            glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql'),
            glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql.gz'),
        );
```

Reply:
I got that solution from Symfony Finder, so maybe we could levarage finder?

Reply:
Jacob has a point, if it is searching by `{.sql,.sql.gz}` pattern, the `GLOB_BRACE` is mandatory to properly work.
Tested locally and returns nothing with the fallback to `0`. 

Maybe 2 glob calls is not that ideal, what about just
```suggestion
        return glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql*');
```
?

Reply:
is it working with that? than its fine.